.github/DEVELOPMENT.md

```diff
@@ -11,51 +11,10 @@ In most cases, when you have Visual Studio installed with the .NET workloads che
 1. Install the latest .NET 6:  
    <!--- [Win (x64)](https://aka.ms/dotnet/6.0.2xx/daily/dotnet-sdk-win-x64.exe)   -->
    - [Install the latest Public Preview of Visual Studio](https://docs.microsoft.com/en-us/dotnet/maui/get-started/installation/)
-   - [macOS (x64)](https://aka.ms/dotnet/6.0.3xx/daily/dotnet-sdk-osx-x64.pkg)  
-   - [macOS (arm64)](https://aka.ms/dotnet/6.0.3xx/daily/dotnet-sdk-osx-arm64.pkg)
-2. Clear your nuget cache:  
-   ```
-   dotnet nuget locals all --clear
-   ```
-   > NOTE: this is going to contain the "stable" versions of the packages, so you will have to clear the NuGet cache when this feed changes and when .NET ships. The various `darc-pub-dotnet-*` feeds are temporary and are generated on various builds. These feeds my disappear and be replaced with new ones as new builds come out. Make sure to verify that you are on the latest here and clear the nuget cache if it changes.
-4. If you're on a Windows development machine, install [SDK 20348](https://go.microsoft.com/fwlink/?linkid=2164145)
-5. If you're on a MacOS development machine, install [PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell-on-macos)
+   - [macOS](https://dotnet.microsoft.com/en-us/download/dotnet/6.0)  
+2. If you're on a Windows development machine, install [SDK 20348](https://go.microsoft.com/fwlink/?linkid=2164145)
+3. If you're on a MacOS development machine, install [PowerShell](https://docs.microsoft.com/powershell/scripting/install/installing-powershell-on-macos)
    
-
-### .NET MAUI Workload
-
-> You'll probably need to run these commands with elevated privileges:
-
-Install the .NET MAUI workload using the versions from a particular branch:  
-
-#### main branch
-
-1. First, install .NET SDK 6.0.300
```

Comment:
@PureWeen just confirming if this means `maui` main has now transitioned to the `6.0.4xx` SDK? Is there any place in the source where the expected SDK version is established?

Reply:
Yes, it's been set here
https://github.com/dotnet/maui/blob/main/eng/Versions.props#L4

SDK version wise we are keeping `main` the same as our `net6.0` branch. So, `main` will be set to whatever SDK version is going to be released with the next service release.  This way we can keep contributors that don't work for MSFT happy :-) 

If users want to make changes that will go into our `.net7.0` releases they can do so on `main` without having to install `.net7.0`. This way contributors can be more successful just using public preview VS.